torch/testing/_internal/common_modules.py

```diff
@@ -2564,7 +2571,16 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
     ModuleInfo(torch.nn.AvgPool2d,
                module_inputs_func=module_inputs_torch_nn_AvgPool2d,
                skips=(
-                   DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),)
+                   DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),
+                   # The difference between channels last backward and
+                   # channels first backward of AvgPool2d on CUDA is too large
+                   # See https://github.com/pytorch/pytorch/issues/107201
+                   DecorateInfo(
+                       unittest.expectedFailure,
+                       'TestModule',
+                       'test_memory_format',
+                       active_if=lambda p: p['training'] and p['device'] == 'cuda',
+                   ),),
```

Comment:
I think `p['device'] == 'cuda'` here might not work and is causing the CI failure because the device will actually be something like `'cuda:0'` (verified this locally). This should work instead

```suggestion
                   DecorateInfo(
                       unittest.expectedFailure,
                       'TestModule',
                       'test_memory_format',
                       active_if=lambda p: p['training'],
                       device_type='cuda',
                   ),)
```